src/components/general/progress-tracking/ProgressDriversVisualization.tsx

```diff
@@ -221,11 +223,13 @@ export function ProgressDriversVisualization({ metric, desiredOutcome, title }:
         trigger: 'axis',
         formatter: (params) => {
           const year = params[0].axisValue;
+          const noDataColor = theme.graphColors.red030 || '#ef9a9a';
+          const noDataText = t('no-data-reported', 'No data reported');
```

Comment:
What does the second argument do here?

```suggestion
          const noDataText = t('no-data-reported');
```